modules/swagger-codegen/src/main/resources/lua/api.mustache

```diff
@@ -4,6 +4,7 @@
 {{#operations}}
 local http_request = require "http.request"
 local http_util = require "http.util"
+local url_encode = require "http.util".encodeURIComponent
```

Comment:
No need to make this a local: call it as `http_util.encodeURIComponent` when needed.